### PR TITLE
Why we built Atlantis blog article

### PR DESCRIPTION
**Pull request by lkysow**
- wrote new blog article
- getting started and faq now link to Github README. Until we fully fill out this part of the website I think it's best to link to the full docs, otherwise it's confusing because there's not much information
- added a link to the blog article from the home page
- deleted example blog article
- add link to blog article from README

@anubhavmishra how do I build the blog once this is checked in? Is it automagic?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ atlantis
 .vscode
 atlantis.db
 output
+website/src/public
```
Comment (lkysow) [inline .gitignore]: is this supposed to be checked in?

File: website/src/content/blog/atlantis-release.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
++++
```
Comment (lkysow) [inline website/src/content/blog/atlantis-release.md]: Here's the blog

File: website/src/content/blog/atlantis-release.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+toc = true
++++
+
+We're very excited to announce the open source release of Atlantis! Atlantis is a tool for collaborating on Terraform that's been in use at [https://hootsuite.com](Hootsuite) for over a year. The core functionality of Atlantis enables developers and operators to run `terraform plan` and `apply` directly from Terraform pull requests. Atlantis then comments back on the pull request with the output of the commands:
+
+<img src="/img/demo-large.gif">
+
+This is a simple feature, however it has had a massive effect on how our team writes Terraform. By bringing a Terraform workflow to pull requests, Atlantis helped our Ops team collaborate better on Terraform and also enabled our entire development team to write and execute Terraform safely.
+
+Atlantis was built to solve two problems that arose at Hootsuite as we adopted Terraform:
```
Comment (lkysow) [inline website/src/content/blog/atlantis-release.md]: I removed the part about environment locking. I think we can write a full blog post about environments. I started on this section, but I wanted to explain how we utilize environments and then it started to get too big. I think without a good intro to how we work with environments just a quick section on environment locking might be confusing.

File: website/src/themes/kube/layouts/index.html [hunk 4]
```diff
@@ -3,40 +3,45 @@
 {{ define "main"}}
 
 <div id="main">
-  <div id="hero">
```
Comment (lkysow) [inline website/src/themes/kube/layouts/index.html]: just formatting

File: .gitignore (hunk 1 above)
Comment (anubhavmishra) [reply .gitignore]: I would like to check it in just for backup and versioning purposes.

File: website/src/content/blog/atlantis-release.md (hunk 3 above)
Comment (anubhavmishra) [reply website/src/content/blog/atlantis-release.md]: cool!

**Pull request merged**